{{cookiecutter.github_project_name}}/MANIFEST.in

```diff
@@ -6,7 +6,7 @@ include pytest.ini
 include .coverage.rc
 
 include package.json
-include lerna.json
+include {{ cookiecutter.python_package_name }}/*.tgz
```

Comment:
Would it not be cleaner to put these in a suitable subdirectory? It would also prevent accidental deletions for the clean step below in case there are other sources of archives in the package root.